charts/kubeai/values.yaml

```diff
@@ -49,6 +49,16 @@ resourceProfiles:
         operator: "Equal"
         value: "present"
         effect: "NoSchedule"
+  nvidia-gpu-h100:
```

Comment:
Can you also add the GKE-specific node selectors to the GKE install guide / the GKE values file?

Other than that - LGTM - feel free to merge

Reply:
done